extensions/wikia/AdEngine/js/template/roadblock.js

```diff
@@ -0,0 +1,38 @@
+/*global define, require*/
+define('ext.wikia.adEngine.template.roadblock', [
+	'ext.wikia.adEngine.context.uapContext',
+	'wikia.log',
+	require.optional('ext.wikia.adEngine.template.skin')
+], function (
+	uapContext,
+	log,
+	skinTemplate
+) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.template.roadblock',
+		uapType = 'ruap';
+
+	/**
+	 * @param {object} params
+	 * @param {string} [params.uap] - UAP ATF line item id
+	 * @param {object} [params.skin] - skin template params (see skin template for more info)
+ 	 */
+	function show(params) {
+		var isSkinAvailable = params.skin && params.skin.skinImage;
+
+		uapContext.setUapId(params.uap);
+		uapContext.setType(uapType);
+
+		if (skinTemplate && isSkinAvailable) {
+			log(['show', 'loading skin', params.skin], log.levels.info, logGroup);
+			skinTemplate.show(params.skin);
+		}
+
+		log(['show', params.uap], log.levels.info, logGroup);
```

Comment:
Did medrec refreshing disappeared for purpose or is it a bug?

Reply:
fixed